test/SafeProtocolRegistry.spec.ts

```diff
@@ -40,30 +43,33 @@ describe("SafeProtocolRegistry", async () => {
 
     it("Should allow only owner to flag a integration", async () => {
         const { safeProtocolRegistry } = await loadFixture(deployContractFixture);
-        await safeProtocolRegistry.connect(owner).addIntegration(AddressZero, IntegrationType.Plugin);
+        const mockHookAddress = await (await getHooksWithPassingChecks()).getAddress();
+        await safeProtocolRegistry.connect(owner).addIntegration(mockHookAddress, IntegrationType.Hooks);
 
-        await expect(safeProtocolRegistry.connect(user1).flagIntegration(AddressZero)).to.be.revertedWith(
+        await expect(safeProtocolRegistry.connect(user1).flagIntegration(mockHookAddress)).to.be.revertedWith(
             "Ownable: caller is not the owner",
         );
 
-        expect(await safeProtocolRegistry.connect(owner).flagIntegration(AddressZero));
+        expect(await safeProtocolRegistry.connect(owner).flagIntegration(mockHookAddress));
 
-        const [flaggedAt] = await safeProtocolRegistry.check.staticCall(AddressZero);
+        const [flaggedAt] = await safeProtocolRegistry.check.staticCall(mockHookAddress);
         expect(flaggedAt).to.be.gt(0);
     });
 
     it("Should allow only owner to flag a integration only once", async () => {
         const { safeProtocolRegistry } = await loadFixture(deployContractFixture);
-        await safeProtocolRegistry.connect(owner).addIntegration(AddressZero, IntegrationType.Plugin);
+        const mockHookAddress = await (await getHooksWithPassingChecks()).getAddress();
```

Comment:
I think in ethers v6 a contract has a `target` property that returns an address which is synchronous

Reply:
Replaced use of `.getAddress()` with `target` property in files changes in this PR